### ISSUE TITLE
pip install tries to install HDF5 into system location

### ISSUE THREAD

**Issue by gshamov** [gersolar/netcdf#23]
This basically should not happen.

**Comment by ecolell** [comment]
@gshamov You are right. I don't taked that into account when I've worked on this. Right now it is impossible to me to fix this. But you are welcome, I could deploy the fix to pypi :+1:.
